src/index.ts

```diff
@@ -4,11 +4,12 @@ import * as ios from "./ios";
 
 export interface Config extends Partial<android.Config>, Partial<ios.Config> {
   icon: Partial<input.Config>;
+  exclude?: { android: boolean; ios: boolean };
```

Comment:
Maybe a bit unnecessary to introduce a separate internal configuration value type. I think this could just as well be the same type as the configuration file value.

Reply:
I thought it would better to already have a `boolean` value for each platform when calling `generate()` and handle reading the values from the array in `cli.ts`. But we could also pass `platforms: Platform[]` directly and handle it there, did you have any specific ideas?

Reply:
I'd say we can go with the same data structure internally as well. Would let the cli module provide the default value though, and perform the input data normalisation there also.